aspects/request_timeout_handler.go

```diff
@@ -0,0 +1,28 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"strconv"
+)
+
+func QueuedRequestTimeout(f httprouter.Handle, custHeaders config.CustomHeaders) httprouter.Handle {
+
+	return func(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+
+		reqTimeInQueue := r.Header.Get(custHeaders.RequestTimeInQueue)
+		reqTimeFloat, _ := strconv.ParseFloat(reqTimeInQueue, 64)
```

Comment:
I like how this is implemented as a wrapper around an endpoint handler! We do need to check for errors here though and default to processing the request if an error parsing either of these headers occurs. Mainly because I'm not sure if this header will even exist for cases where the LB doesn't need to queue the request, so add an error check here just in case something goes wrong.

Reply:
Good catch! Fixed!